src/Icon/Icon.js

```diff
@@ -54,9 +54,10 @@ const iconStyles = ({ color, rtl, size, theme: baseTheme }) => {
 const Root = createStyledComponent('svg', iconStyles, { rootEl: 'svg' });
 
 /**
- * The Icon component displays a graphic symbol representing an object or function.
+ * Icons use graphical symbols to represent an object or concept in your UI.
+ * They can be used to aid comprehension of core actions in your app, and to provide feedback for user input.
  *
- * The package provides a large number of prepackaged icon components as well
+ * Mineral UI provides a large number of bundled icon components as well
  * as a generic icon component to which you can provide your own SVG.
```

Comment:
icon ➡️  Icon

Reply:
addressed

Reply:
You got one of these, but missed the second instance in this sentence.